lib/update-lockfile.js

```diff
@@ -32,7 +32,9 @@ module.exports = function updateLockfile (dependency, options) {
 
     if (options.yarn) {
       exec(`yarn add ${args}`)
-    } else {
+    }
+
+    if (options.npm) {
```

Comment:
@timbru31 I just noticed the `package-json.lock` was changed. Is there a particular reason for that?